src/Umbraco.Core/Constants-PropertyEditors.cs

```diff
@@ -173,7 +173,7 @@ public static class Aliases
             /// <summary>
             ///     TinyMCE
             /// </summary>
-            public const string TinyMce = "Umbraco.TinyMCE";
+            public const string TinyMce = "Umbraco.RichText";
```

Comment:
Should we rename the name of the constant `TinyMce`, as well?

Reply:
Hmmm that would be breaking, maybe we should Create a new Constant called `RichText`, and then obsolete this one ? 🤔 

So something like:
```
public const string RichText = "Umbraco.RichText";

[Obsolete("please use RichText constant instead")]
public const string TinyMce = RichText;
```

Reply:
That would be ideal, yeah!